wolfcrypt/src/sp_int.c

```diff
@@ -13460,6 +13460,9 @@ int sp_gcd(sp_int* a, sp_int* b, sp_int* r)
     if ((a == NULL) || (b == NULL) || (r == NULL)) {
         err = MP_VAL;
     }
+    else if (a->size >= SP_INT_DIGITS || b->size >= SP_INT_DIGITS) {
```

Comment:
used not size

Reply:
Just saw this, silly mistake on my side thanks Sean. Will fix it.